DQM/L1TMonitor/src/L1TStage2uGTCaloLayer2Comp.cc

```diff
@@ -346,8 +346,7 @@ bool L1TStage2uGTCaloLayer2Comp::compareSums(
 
   // if either calol2 or ugt collections are empty, or they have different
```

Comment:
Please adapt also the comment to the new code.

Reply:
Thanks for pointing this out Thomas, I modified to reflect the code and updated the PR.